samples/react-application-breadcrumb/src/extensions/siteBreadcrumb/loc/myStrings.d.ts

```diff
@@ -2,7 +2,7 @@ declare interface ISiteBreadcrumbStrings {
   Title: string;
 }
 
-declare module 'siteBreadcrumbStrings' {
+declare module 'SiteBreadcrumbApplicationCustomizerStrings' {
```

Comment:
Unfortunately, after this change we should also update `config/config.json` and change the `localizedResources` to the new name as well

currently without this change if I run the `gulp bundle --ship` it will crash

![image](https://github.com/user-attachments/assets/b5ea2e34-ec2f-4e3a-a8b1-bfef62007da3)

Reply:
Not sure how that happened because I installed it and tested it. I've just put the name back to the original, checked the build and deployed it out to my test site to ensure it was still working. I cleared everything out and re-added it to ensure it was fresh. 

![image](https://github.com/user-attachments/assets/7f953ea8-196e-4b7e-9634-0801298dc7c0)